mlos_bench/mlos_bench/services/remote/azure/azure_services.py

```diff
@@ -145,7 +145,8 @@ def __init__(self, config: dict, parent: Service):
         self._poll_timeout = float(config.get("pollTimeout", AzureVMService._POLL_TIMEOUT))
         self._request_timeout = float(config.get("requestTimeout", AzureVMService._REQUEST_TIMEOUT))
 
-        self._deploy_template = self.config_loader_service.load_config(config['deployTemplatePath'])
+        # TODO: Provide external schema validation?
+        self._deploy_template = self.config_loader_service.load_config(config['deployTemplatePath'], schema_type=None)
```

Comment:
I am afraid that call removes `$schema` from ARM Template, and, as a result, our call to Azure API fails. We need to fix this

Reply:
Changed the loader logic to only remove the `$schema` attribute if the `schema_type is not None`.